src/custom-elements/src/components/carousel/carousel.tsx

```diff
@@ -383,29 +392,46 @@ export class Carousel {
   }
 
   _getItemVisibility(index) {
-    let minActiveItem = this.view * this.maxItemsShown;
-    const maxActiveItem = minActiveItem + this.maxItemsShown;
+    let minVisibleItem = this.view * this.maxItemsShown;
+    const maxVisibleItem = minVisibleItem + this.maxItemsShown;
 
     if (this.isLastView && this.restItemsShown > 0) {
-      minActiveItem =
-        minActiveItem - (this.maxItemsShown - this.restItemsShown);
+      minVisibleItem =
+        minVisibleItem - (this.maxItemsShown - this.restItemsShown);
     }
 
-    return index >= minActiveItem && index < maxActiveItem;
+    return index >= minVisibleItem && index < maxVisibleItem;
   }
 
-  setActiveClass() {
+  setVisibleClass() {
     const carouselItems = this._getCarouselItems();
 
     carouselItems.forEach((item, index) => {
       const isVisible = this._getItemVisibility(index);
 
-      item.classList.toggle(ACTIVE_CLASS, isVisible);
+      item.classList.toggle('-visible', isVisible);
     });
   }
 
+  _getStatComponent(): HTMLElement {
+    return this.el.querySelector(`.${STAT_CLASSES.STAT}`) as HTMLElement;
+  }
+
+  _getGap(): number {
+    const statComponent = this._getStatComponent();
+
+    if (statComponent) {
+      const statComputedStyle = window.getComputedStyle(statComponent);
+      const gap = parseInt(statComputedStyle.gap, 10);
+
+      return gap;
```

Comment:
SQ might complain about this:
```suggestion
      return parseInt(statComputedStyle.gap, 10);
```